components/external_server/src/database/reports.py

```diff
@@ -56,7 +56,8 @@ def _get_change_key(change: Change) -> str:
 
 def metrics_of_subject(database: Database, subject_uuid: SubjectId, max_iso_timestamp: str = "") -> list[MetricId]:
     """Return all metric uuid's for one subject, without the entities, except for the most recent one."""
-    report_filter = {f"subjects.{subject_uuid}": DOES_EXIST}
+    report_filter: dict[str, bool | dict[str, bool] | dict[str, str]] = {}
```

Comment:
This is just to be taken at face value when there's no context, what's going on? :D

Reply:
MongoDB filters are a beast :-)  What can I add to help the reader here? A link to MongoDB docs?

Reply:
some inline comments and link to docs will probably suffice